src/action/index.js

```diff
@@ -34,7 +34,7 @@ export const log = new LogAction(store, ipc);
 export const nav = new NavAction(store);
 export const grpc = new GrpcAction(store, ipc);
 export const notify = new NotificationAction(store, nav);
-export const wallet = new WalletAction(store, grpc, db, nav, notify);
+export const wallet = new WalletAction(store, grpc, db, nav, notify, ipc);
```

Comment:
We shouldn't inject electron's ipc into actions directly as this would make porting to mobile harder. Rather we should use the grpc actions to wrap ipc calls.

Reply:
true, fixed!